ideas/1-offline-inboxing.md

```diff
@@ -0,0 +1,132 @@
+
+Uncertainty: Swarm availability, both w.r.t. other ideas and holidays (seems OK). Availability normal for 2 weeks: Adam 2w 22 Dec, Eugene, Boris, Dmitry, Oskar After Christmas no more Go dev necessary.
+
+### Iteration 1.5/2 - Joe Blockchain goes to town (multiple wnodes)
+Goal Date: TBD - est. +~2w, due to holidays possibly delayed till mid January
+State: Aborted
+Completed: 2018-01-14 (very partial; read only screen under flag)
+Description: Ability to toggle between different wnodes in the app. Show basic connectivity to wnode in interface. The wnodes identifiers are hardcoded in the app. Add by enode (?). Multiple wnodes enabled for redundancy (?).
+
+Uncertainty: Go dev work necessary. Seems docs and options for wnode-status but other than that not. Only on retainer.
+
+### Archive (old stuff)
+
+#### Design track
+
+Consider how to surface and solve the following challenges in our UI. wnode and mailbox used interchangeably. This list should probably be split into MVP concerns and later concerns as there is quite a lot.
+
+- How will user select and deselect a wnode?
+- How do we communicate more "trusted" nodes, like the ones we operate initially?
+- How do we communicate and quantify reputation such as uptime and reliability? Start ad hoc (upvotes? number of connected nodes? "official" Status nodes with guaranteed uptime?)
+- How will the cost of using node be determined? Initial setup and running costs?
+- What happens if we fail to connect to our mailbox? How do we communicate this in a chat?
+- How communicate to the sender that a message was ACKed by a mail box but not by end user?
+- What about if a user wants to use multiple mailboxes?
+- Where can a user get an overview of the mail server / contracts they are using?
+- How will ongoing payment signing work?
+- How do we communicate a certain wnode is dead or can't be trusted for some reason?
+- How do we communicate capacity of a wnode to ensure quality of service?
+- How does a wnode operator accept or deny a request to act as mail server? How is this communicated to end user?
+- Initially a wnode will probably just be a CLI thing. How do we surface options like how much compensation we want? And what about when this can be run through Status Desktop? How do you decide how much capacity you have as a wnode?
+
+#### Iteration 1..N
+
+TBD. Smorgasbord of desirable features. Revisit this.
+
+- Discovery of wnodes
+- Ability to select wnodes
+- Running wnodes in cluster and making it possible for others to run them
+- Giving feedback in chat about state of messages
+- Using multiple wnodes
+- Cryptoeconomics to incentive running of nodes
+- Basic reputation system to signal quality of service
+
+Goal Date:
+Description:
+
+## Supporting Role Communication
```

Comment:
Missing success metric and exit criteria. Not part of requirements for ideas at the time. Probably useful to add something ad-hoc here.